qiskit_experiments/library/calibration/rabi.py

```diff
@@ -103,7 +116,15 @@ def _default_analysis_options(cls) -> Options:
 
         return options
 
-    def __init__(self, qubit: int):
+    def __init__(
+        self,
+        qubit: int,
+        cals: Optional[Calibrations] = None,
+        schedule_name: Optional[str] = "x",
+        cal_parameter_name: Optional[str] = "amp",
```

Comment:
I think adding `self.calibration_options`, `self._default_calibration_options`, `self._set_calibration_options` to the base calibration class is reasonable, i.e. user can easily find calibration-relevant parameters, developer can easily give (inheritable) default values.

Reply:
This seems reasonable to me because some of these calibration experiments are getting to have a lot of experiment options. On the other hand, there may be some options that could go either way where there could be confusion about what type an option is.

Reply:
Good point. Done here: 5c45d54